modules/reindex/src/test/java/org/elasticsearch/index/reindex/RoundTripTests.java

```diff
@@ -120,9 +121,24 @@ public void testBulkIndexByScrollResponse() throws IOException {
         assertResponseEquals(response, tripped);
     }
 
+    public void testRethrottleRequest() throws IOException {
+        RethrottleRequest request = new RethrottleRequest();
+        request.setRequestsPerSecond((float) randomDoubleBetween(0, Double.POSITIVE_INFINITY, false));
```

Comment:
Weird. I remember writing this and thinking I used Float.POSITIVE_INFINITY. But I didn't. The weird thing is that this passes. A lot. I guess I just got lucky. 3d50d17